docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,35 +1,372 @@
+
+#### Import an existing project
+
+To import an existing project, you can use Dagster's GitHub app or the dagster-cloud CLI.
+
+<TabGroup>
+<TabItem name="GitHub app">
+
+**Using the GitHub app**
+
+Using the GitHub app to import an existing project also sets up CI/CD for you.
+
+Before you get started, note that your project must have a `dagster_cloud.yaml` file in order for Dagster Cloud to deploy it. If your project doesn't contain this file, you'll need to add it before proceeding.
+
+In the root of your project, create a `dagster_cloud.yaml` with the following code:
+
+```yaml
+locations:
+  - location_name: my_location_name # the name of the code location
+    code_source:
+      package_name: hackernews # the name of the python package associated with the code location
+```
+
+After you've committed the file to the repository, come back to Dagster Cloud to complete the import process:
+
+1. Click the **Import an existing project** tab.
+2. Sign in to your GitHub account, if prompted.
+3. In Dagster Cloud, define the following:
+   - **Git scope** - Select the organization or account that the repository is in.
+   - **Repository** - Select the repository.
+4. Click **Deploy**.
+
+When finished, [continue to the next step](#step-4-set-up-environment-variables-and-secrets).
+
+</TabItem>
+<TabItem name="dagster-cloud CLI">
+
+**Using the dagster-cloud CLI**
+
+<Note>
+  Using the dagster-cloud CLI requires a recent version of Python 3 and Docker.
+</Note>
+
+To complete this step using the CLI, you can use your own Dagster code or the [Dagster Cloud Serverless Quickstart](https://github.com/dagster-io/dagster-cloud-serverless-quickstart). The quickstart is a template with everything you need to get started using Serverless in Dagster Cloud, including CI/CD configuration and the required `dagster_cloud.yaml` file.
+
+- **If using the template**, click [**Use this template** in the repository](https://github.com/dagster-io/dagster-cloud-serverless-quickstart). Finish setting up the repository on GitHub and clone it to your local environment.
+
+- **If not using the template**, add `dagster-cloud` as a dependency in `setup.py`. [Click here for an example](https://github.com/dagster-io/dagster-cloud/blob/main/dagster-cloud-examples/setup.py). This is already done for you if using the Hybrid quickstart template.
+
+After you've finished setting up your local project, move on to deploying using the CLI:
+
+1. To install the dagster-cloud CLI, run:
+
+   ```shell
+   pip install dagster-cloud
+   ```
+
+2. Next, you'll need to authenticate to Dagster Cloud. Run the following command and follow the prompts to log in:
+
+   ```shell
+   dagster-cloud config setup
+   ```
+
+3. After you've successfully authenticated, run the following to deploy your code to Dagster Cloud:
+
+   ```shell
+   dagster-cloud serverless deploy \
+         --location-name "<LOCATION_NAME>" \
+         --package-name <PACKAGE_NAME> \
```

Comment:
worth adding the comments about what location-name and package-name are here too, because the comments were only in the "Using the GitHub app" tab and people land here may not see the content in the other tab

Reply:
Good point! Updated